### DISCUSSION TITLE
iOS compatibility

### DISCUSSION THREAD

**Discussion by aldaw** [vslinko/obsidian-outliner#196] [Archive]
Since your add-on is written basically only in JS/TS shouldn't that work on iOS to?
Do you think that there is anything which can be done from this git branch to make it compatible?
Or is it a limitation from the iOS Obsidian app it self? :)

I hope everything is amazing at your side, and thank you for any second you have put in to work with your genius fingers!

PS.: people love your add-on! and missing it heavily on iOS :P - https://www.reddit.com/r/ObsidianMD/comments/oydidm/ios_app_plugin_outliner/

**Comment by vslinko** [comment]
@aldaw Hi! Thank you for your feedback!

Obsidian Mobile is unsupported yet. Please follow #124 for more information.
